tests/DCPS/Compiler/char_literals/CMakeLists.txt

```
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.8...3.27)
+project(opendds_compiler_char_literals CXX)
+enable_testing()
+
+find_package(OpenDDS REQUIRED)
+include(opendds_testing)
+
+set(target_prefix "${PROJECT_NAME}_")
+set(src "${CMAKE_CURRENT_SOURCE_DIR}")
+set(dst ${CMAKE_CURRENT_BINARY_DIR})
+set(opendds_libs
+  OpenDDS::Dcps
+)
+
+# IDL Library
+set(idl "${target_prefix}idl")
+add_library(${idl})
+opendds_target_sources(${idl} PUBLIC "${src}/test.idl" OPENDDS_IDL_OPTIONS "-Lc++11")
+target_link_libraries(${idl} PUBLIC OpenDDS::Dcps)
+set(idl_dir "${dst}")
```

Part of using `run_test.pl`.

```suggestion
set(idl_dir "${dst}/cpp11")
```